### DISCUSSION TITLE
Remote NVR as a source

### DISCUSSION THREAD

**Discussion by yodaphone** [datarhei/restreamer#558] [Q&A]
Hi

I have a NVR with 8 cameras. Is there a way to add these cameras as a source?

i enabled RTSP and when i connect to it, i only get one camera. is there a way to connect to other cameras?

**Comment by svenerbeck** [comment]
Dear @yodaphone

Thank you so much for reaching out with your question about the connection of multiple cameras to your Network Video Recorder (NVR).

The answer is „Yes." You can effectively add as many sources as your hardware capacity, and bandwidth allows.

When connecting to multiple cameras, it's important to note that each camera on your NVR will have a unique RTSP (Real-Time Streaming Protocol) stream URL. This means that to connect to different cameras. You must use the specific RTSP URL for each one.

Upon enabling RTSP, if you can only connect to one camera, you're likely using the RTSP URL for that specific camera. You'll need their unique RTSP URLs to connect to the other cameras.

Please review your NVR and camera documentation to understand how to derive these unique URLs. Typically, they can be found in the settings of each camera. Once you have these URLs, you can add them as individual sources.

If you need more help or clarification, please don't hesitate to contact us. We are here to help.

Cheerio
Sven

PS: IIRC, you are on Discord too - Feel free to contact me PM.
